README.md

```diff
@@ -1,11 +1,87 @@
 # Topiary
 
-![CI Status](https://github.com/tweag/topiary/actions/workflows/ci.yml/badge.svg)
+![CI Status][ci-badge]
 
 Topiary aims to be a uniform formatter for simple languages, as part of
-the [Tree-sitter](https://tree-sitter.github.io/tree-sitter/) ecosystem.
-It is named after the art of clipping or trimming trees into fantastic
-shapes.
+the [Tree-sitter] ecosystem. It is named after the art of clipping or
+trimming trees into fantastic shapes.
+
+## Motivation
```

Comment:
Before even this section we need a section explaining to the reader in which case he should care about Topiary. Something brief and to the point. There are two typical users: formatter authors, and formatter users. The value proposition is easier to articulate for authors, evidently. For users, we can explain the formatting philosophy (which we describe in more details below), and maybe the fact that it can conveniently serve as a formatter for many languages in their repository (ore repositories even).

Reply:
Added in fd18ebc1c1a